theseus_gui/src-tauri/Cargo.toml

```diff
@@ -29,6 +29,8 @@ daedalus = {version = "0.1.15", features = ["bincode"] }
 url = "2.2"
 uuid = { version = "1.1", features = ["serde", "v4"] }
 
+os-version = "0.2.0"
```

Comment:
would use os_info instead, has a similar API surface but os info seems to support more stuff